presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -65,11 +65,13 @@ Date and Time Functions
 
 .. function:: from_iso8601_timestamp(string) -> timestamp with time zone
 
-    Parses the ISO 8601 formatted ``string`` into a ``timestamp with time zone``.
+    Parses the ISO 8601 formatted date and time ``string``, such as
+    ``2020-05-11T11:15:05.055-08:00``, into a ``timestamp with time zone``.
```

Comment:
Let's write these as example queries with the output, like we do for other functions.

Reply:
ok

Reply:
Done now ..